dkpro-core-api-parameter-asl/src/main/java/de/tudarmstadt/ukp/dkpro/core/api/parameter/ResourceParameter.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.dkpro.core.api.parameter;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface ResourceParameter {
+    /**
+     * The MIME types supported by this component.
+     *
+     * @return the supported MIME types
+     */
+    String[] value() default NO_DEFAULT_VALUE;
+
+    /**
+     * Provides the default value that tells the ResourceParameter that no value has been provided.
+     */
+    public static final String NO_DEFAULT_VALUE = "org.apache.uima.fit.descriptor.MimeTypeCapability.NO_DEFAULT_VALUE";
```

Comment:
That would be `de.tudarmstadt.ukp.dkpro.core.api.parameter.NO_DEFAULT_VALUE`, right? However, I would tend towards using `{}` as the default value in line 33 and not using this uimaFIT-ismic hack.